pkg/snapshot/overlay.go

```diff
@@ -885,31 +885,21 @@ func (o *snapshotter) identifySnapshotStorageType(id string, info snapshots.Info
 
 	}
 
+	// check overlaybd.commit
 	filePath := o.magicFilePath(id)
-
-	f, err := os.Open(filePath)
-	if err != nil {
-		if os.IsNotExist(err) {
-			return storageTypeNormal, nil
-		}
-		return storageTypeUnknown, errors.Wrapf(err, "failed to open %s", filePath)
+	st, err := o.identifyLocalStorageType(filePath)
```

Comment:
what if err != nil ?

Reply:
It will try the next step. If it fails to open o.magicFilePath(id) in identifyLocalStorageType(), an error will be returned. And this error may has nothing to do with identifying a writable local blob.

Reply:
Ok. Maybe we can add debug log here :)

Reply:
Yes, a debug log is added here.